src/transformers/models/dpt/configuration_dpt.py

```diff
@@ -96,6 +96,8 @@ class DPTConfig(PretrainedConfig):
             Whether to use bias in the pre-activate residual units of the fusion blocks.
         add_projection (`bool`, *optional*, defaults to `False`):
             Whether to add a projection layer before the depth estimation head.
+        use_size (`bool`, *optional*, defaults to `False`):
+            Whether to use the size for interpolation, rather than scale factor, as in Depth Anything.
```

Comment:
Let's use a better name here. From the name, it's not clear what `size` is being used from, it also doesn't extend well. At the moment, we're assuming that if we're not using `size`, then we're using scale factor, but what if a new model uses a different param? 

I'd suggest a parameter where one can select `"size"` or `"scale_factor"` as options